Dockerfile

```diff
@@ -28,18 +28,30 @@ ARG STRIPE_API_KEY
 ARG STRIPE_WEBHOOK_SIGNING_SECRET
 ARG PHX_SERVER
 ARG DATABASE_URL
+ARG PLATFORM=amd64
 
-ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
+ARG BUILDER_IMAGE="hexpm/elixir-${PLATFORM}:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
 ARG RUNNER_IMAGE="debian:${DEBIAN_VERSION}"
 
-FROM --platform=linux/amd64 ${BUILDER_IMAGE} as builder
+FROM --platform=linux/${PLATFORM} ${BUILDER_IMAGE} as builder_amd64
```

Comment:
I will still let builder because the platform can change

Reply:
Not sure I understand this, you mean if you build for 2 platforms you don't want name conflicts?
Why put `builder_amd64` as it can be different?